deployment/scripts/.env

```diff
@@ -1,5 +1,6 @@
 # Kubernetes Provider Enviromment Variables
 k8s_provider="eks" # choices are eks | other
+TF_VAR_eks_cluster_name="fuel-core-deploy"
```

Comment:
Looks like this is no longer used?

Reply:
its being used by the all the scripts in fuel-core and infrastructure- gives the EKS name-

https://github.com/FuelLabs/fuel-core/blob/master/deployment/scripts/fuel-core-deploy.sh#:~:text=aws%20eks%20update%2Dkubeconfig%20%2D%2Dname%20%24%7BTF_VAR_eks_cluster_name%7D

Need it to authenticate to EKS cluster